lib/gateway/Shard.js

```diff
@@ -2321,11 +1994,8 @@ class Shard extends EventEmitter {
                 this.emit("stageInstanceDelete", guild.stageInstances.remove(packet.d) || new StageInstance(packet.d, this.client));
                 break;
             }
-            case "MESSAGE_ACK": // Ignore these
-            case "GUILD_INTEGRATIONS_UPDATE":
-            case "USER_SETTINGS_UPDATE":
-            case "CHANNEL_PINS_ACK": {
-                break;
+            case "GUILD_INTEGRATIONS_UPDATE": {
+                break; // REVIEW
```

Comment:
This sends a guildID on twitch/youtube integration update, there are some other integration events on the discord docs but they seem to not be sent

Reply:
We ignored these for idk how long. Am asking abal for clarification (wish me luck for a reply)

Reply:
So it seems like when it was first implemented it basically did nothing, I'm going to unignore this event